gen/build_deploy/bash/dcos_generate_config.sh.in

```diff
@@ -50,4 +55,20 @@ else
     trap 'docker kill {genconf_tar}' HUP QUIT INT TERM
     docker run --rm --name=$DCOS_INSTALLER_CONTAINER_NAME -i -p $PORT:9000 -v $(pwd)/genconf/:/genconf {docker_image_name} "$@"
 fi
-exit $?
+
```

Comment:
Does this work if `$DCOS_INSTALLER_DAEMONIZE" != "true"`?

Reply:
@jongiddy if `DCOS_INSTALLER_DAEMONIZE  == "true"`, then `-d genconf/ca` will most likely evaluate to `false`, because gen would not block during extraction.  That would mean that the CA service would not start, resulting in a bootstrap failure if `exhibitor_tls_required` was `true`. When I was originally looking into this, I couldn't find any documentaion on DCOS_INSTALLER_DEAMONIZE .. any idea what it is used for?